### ISSUE TITLE
fluentd-before.png seems really dishonest

### ISSUE THREAD

**Issue by anarcat** [fluent/fluentd-website#90]
the diagram of "fluentd-before" is really dishonest: it shows a complicated graph of dependencies between apache, syslog, android and other tools, with arrows all over, which is fair. But then it goes totally over the top by drawing one loooong wiggle line that hides the actual architecture and makes it look needlessly complicated.

![image](https://user-images.githubusercontent.com/796623/58963981-1f293700-877c-11e9-8bb3-60277eac82ce.png)

![image](https://user-images.githubusercontent.com/796623/58963993-24868180-877c-11e9-9e9d-516859a35161.png)

If fluentd really solves problems with the older infrastructure, it shouldn't need such artifice to justify its existence.

I would have fixed the image itself (or copy-pasted the long wiggle over to the right graph myself) but the image is a PNG, not an SVG, which makes it much harder to edit. :p

I assume someone somehwere has the original vector version however...

**Comment by repeatedly** [comment]
Yes. Above image is created some years ago. So maybe, original file is gone...
